pkg/ui/src/Toast/Toast.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import { fade } from 'svelte/transition';
+
+	import { X } from 'lucide-svelte';
+
+	import { ToastType } from './enums';
+
+	const dispatch = createEventDispatcher();
+
+	export let type: ToastType;
+	export let dismissible = true;
+
+	const colourLookup = {
+		[ToastType.Error]: 'bg-red-50 text-red-900',
+		[ToastType.Success]: 'bg-green-50 text-green-900'
+	};
+</script>
+
+<div
+	class={`flex items-center justify-between max-w-fit px-4 py-2 gap-x-4 shadow-md rounded-lg ${colourLookup[type]}`}
+	role="alert"
+	transition:fade
+>
+	<p class="text-left text-base font-normal">
+		<slot />
+	</p>
+	{#if dismissible}
+		<button
+			type="button"
+			aria-label="Close toast"
+			class="rounded-lg p-1 text-gray-400 hover:text-gray-800 focus:outline-none focus:outline-gray-700"
```

Comment:
What is achieved with ` focus:outline-none focus:outline-gray-700`? Seems contradictory, or am I missing some a11y quirk?

Reply:
Have experimented with this recently:`focus:outline-none` is required to remove default browser focus. Without it the `outline-gray-700` won't be applied

Reply:
I still get that ugly ring (at least in stories)...I know you've also focused on a11y so idk if that's there on purpose _(I'm using Chrome)_

Reply:
Oooh...I get the ring on click, so it's probably `active:` not `focus:`. I've tried to remove it using `active:outline-none`, to no avail, so it might be the same thing you've described for `focus:`

Reply:
I've been focussing mainly on keyboard usage here and in Edible. I'm using Firefox and it's gone for me on both tab and click focus. I'll have a further read into sorting it on Chrome